packages/calypso-products/src/plans-list.tsx

```diff
@@ -1194,8 +1209,14 @@ const getPlanBusinessDetails = (): IncompleteWPcomPlan => ( {
 	getBlogAudience: () => i18n.translate( 'Best for small businesses' ),
 	getPortfolioAudience: () => i18n.translate( 'Best for small businesses' ),
 	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
-	getPlanTagline: () =>
-		i18n.translate( 'Unlock the power of WordPress with plugins and cloud tools.' ),
+	getPlanTagline: ( flowName ) => {
```

Comment:
I'm having some second thoughts about this change. Sorry! :/ 

Let me come back to this with a clear mind, but it might be the case we'll be better off creating a separate method, something like `getNewsletterPlanTagline`. This is how it's done for other things here, and it may be easier to port to API-fetched properties later. WDYT? 🤷

Reply:
Interesting. We already have `getNewsletterSignupFeatures()` so this might make the call sites more consitent. I don't have a strong opinion on this though, so I'll defer to @escapemanuele to decide :)

Reply:
Another solution could be to have a more generic `getFlowPlanTagline` that accepts a `flowName` property. I was thinking at when we'll want to do the same thing for the Link In Bio flow and others.
But `getNewsletterPlanTagline` is totally fine for me, and it solves our current problem. 

Also, where should the call to `getNewsletterPlanTagline` be? I imagine not in the Plans structure, as to not have the `isNewsletterFlow` function there

Reply:
My thinking was it might be easier to migrate individual/dumb methods to API-fetched values. But, like you said, the conditioning will change, and we'll likely end up refactoring more work there. So, as you like, and happy to leave as-is.

Reply:
My thinking was it might be easier to migrate individual/dumb methods to API-fetched values. But, like you said, the conditioning will change, and we'll likely end up refactoring more work there. So, as you like, and happy to leave as-is.